cmake/projects/SDL2/hunter.cmake

```diff
@@ -0,0 +1,24 @@
+if(DEFINED HUNTER_CMAKE_PROJECTS_SDL2_HUNTER_CMAKE_)
```

Comment:
Header guards may lead to wrong configuration in some cases see wiki:
* https://github.com/ruslo/hunter/wiki/usr.adding.new.package#03-add-new-package

please add comment `# !!! DO NOT PLACE HEADER GUARDS HERE !!!`